### PR TITLE
feat(eigen-client-extra-features): Move inclusion logic

### PR DESCRIPTION
**Pull request by gianbelinche**
## What ❔
This PR moves inclusion and verify logic from disperse blob to get inclusion data

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zkstack dev fmt` and `zkstack dev lint`.

### CODE REVIEW

**Review by juan518munoz — COMMENTED** [review]
[Unused import: `DisperserBlobReply`](https://github.com/lambdaclass/zksync-era/pull/357/files#diff-50639b5d8317ac6c2332a1684199ec9498f9afa4cdb62d959c2e7e3dc733ecf7R25)

**Review by juan518munoz — APPROVED** [review]

File: core/node/da_clients/src/eigen/client.rs [hunk 1]
```diff
@@ -32,6 +32,11 @@ impl EigenClient {
             client: Arc::new(client),
         })
     }
+
+    pub async fn get_commitment(&self, blob_id: &str) -> anyhow::Result<String> {
```
Comment (juanbono) [inline core/node/da_clients/src/eigen/client.rs:36]: same here, anyhow should not be used

File: core/node/da_clients/src/eigen/sdk.rs [hunk 2]
```diff
@@ -155,18 +132,28 @@ impl RawEigenClient {
         let disperser::authenticated_reply::Payload::DisperseReply(disperse_reply) = reply else {
             return Err(anyhow::anyhow!("Unexpected response from server"));
         };
+        Ok(hex::encode(disperse_reply.request_id))
+    }
 
-        // 5. poll for blob status until it reaches the Confirmed state
+    pub async fn get_inclusion_data(&self, blob_id: &str) -> anyhow::Result<String> {
```
Comment (juanbono) [inline core/node/da_clients/src/eigen/sdk.rs:138]: same here

**Review by juanbono — CHANGES_REQUESTED** [review]
Replace anyhow with thiserror

**Review by juanbono — APPROVED** [review]
Please move from anyhow to thiserror in another PR

**Pull request merged**